gradle/libs.versions.toml

```diff
@@ -20,7 +20,9 @@ javaPoet = "1.+"
 javaxInject = "1"
 jansi = "2.+"
 jimfs = "1.+"
-jline = "3.+"
+# 3.25.1 doesn't work with native image
+# https://github.com/jline/jline3/pull/959
```

Comment:
```suggestion
```

Reply:
This fix is definitely required. It may not be sufficient.